invenio_indexer/tasks.py

```diff
@@ -11,35 +11,54 @@
 from celery import shared_task
 
 from .api import RecordIndexer
-
+from .proxies import current_indexer_registry
 
 @shared_task(ignore_result=True)
-def process_bulk_queue(version_type=None, es_bulk_kwargs=None):
+def process_bulk_queue(
```

Comment:
method signatures and indexer instantiation could be simplified if we decide to break compatibility.